types/defines/workflows.d.ts

```diff
@@ -1,13 +1,11 @@
-/**
- * NonRetryableError allows for a user to throw a fatal error
- * that makes a Workflow instance fail immediately without triggering a retry
- */
 declare module "cloudflare:workflows" {
-  export abstract class NonRetryableError extends Error {
-    /**
-     * `__brand` is used to differentiate between `NonRetryableError` and `Error`
-     * and is omitted from the constructor because users should not set it
-     */
+  /**
+   * NonRetryableError allows for a user to throw a fatal error
+   * that makes a Workflow instance fail immediately without triggering a retry
+   */
+  export class NonRetryableError extends Error {
+    // `__brand` is used to differentiate between `NonRetryableError` and `Error`
```

Comment:
What is `__brand` and why does it merit mentioning here?

Reply:
Things like js rpc that are based on v8 serialization do not currently handle custom class-based Error objects very well. With `Error` objects only the core properties like `name`, `message`, and `stack` are handled out of the box. I'm not sure if `__brand` should be mentioned here but I believe the original intent was for brand to be used to signal that this is not a regular error. Unfortunately, v8 won't serialize it either way.

Reply:
done